packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -343,16 +343,12 @@ export class Combobox
   //--------------------------------------------------------------------------
 
   @Listen("click", { target: "document" })
```

Comment:
@jcfranco do you think it would be worthwhile to refactor this event to only be added once for all combobox components? Currently, each combobox sets up an event listener on the document. Could be enhanced to create a single event listener for all combobox components.

Reply:
Not sure. We'd still need to add a mechanism to 1. ensure a single window/doc listener is set up and 2. manage components to be processed by this single callback, no?

Reply:
Correct. It may not be beneficial if there aren't many comboboxes used in a document.